src/GeositeFramework/js/Plugin.js

```diff
@@ -32,7 +32,40 @@ require(['use!Geosite',
                 pluginName = model.get('pluginSrcFolder'),
                 $uiContainer = model.get('$uiContainer'),
                 $legendContainer = model.get('$legendContainer'),
-                logger = new Logger(pluginName);
+                logger = new Logger(pluginName),
+                resizers = {
+                    adjustBy: function(val) {
+                        var currentWidth = parseFloat($uiContainer.css("width"));
+                        var d = $.Deferred();
+                        d.then(function() {
+                            $uiContainer.css({ width: currentWidth + val });
+                        });
+                        return d.resolve();
+                    },
+                    setWidth: function(val) {
+                        var d = $.Deferred(),
+                            setter = function(w) {
+                                return function() {
+                                    $uiContainer.css({ width: "" });
+                                    $uiContainer.removeClass(function() {
+                                        var classes = this.className.split(" ");
+                                        return _.reject(classes, function(c) {
+                                            return /sidebar-width-.*/.test(c);
+                                        });
+                                    });
+                                    $uiContainer.addClass("sidebar-width-" + w);
+                                };
+                            };
+                        if (typeof (val) === "string") {
+                            d.then(setter(val));
```

Comment:
Consider whitelisting the text arguments which are acceptable to this function. I can imagine someone making a small typo such as `setWidth('Large')` and becoming confused as to why nothing happens.